src/components/app/Home.js

```diff
@@ -108,12 +109,12 @@ type HomeProps = {
   retrieveProfileFromFile: File => void,
 };
 
-class Home extends PureComponent {
-  state: {
-    isDragging: boolean,
-    isAddonInstalled: boolean,
-  };
-  props: HomeProps;
+type HomeState = {
+  isDragging: boolean,
+  isAddonInstalled: boolean,
+};
```

Comment:
Note: I could use `{| ... |}` here. There is a global inconsistency in this patch around this, this comes from the fact the patch spans 2 months. I think this can be made more coherent over time, but please tell me if you prefer that I change the diff-ed ones in this patch.

Reply:
I'm optimizing for landing this work sooner. We can do follow-ups for consistency.

Reply:
yep that's also my take :)